topostats/grains.py

```diff
@@ -42,6 +42,7 @@ def __init__(
         absolute_smallest_grain_size: float = None,
         background: float = 0.0,
         base_output_dir: Union[str, Path] = None,
+        zrange: list = None
```

Comment:
Could you please add a doc-string under `Parameters` for what this option is. That way users know and its included in the auto-generated Sphinx documentation which is built from the docstrings automatically.

Reply:
yep, done!